build/build.sh

```diff
@@ -51,7 +51,7 @@ build_source()
   mvn clean install ${MVN_CONFIG}
 }
 
-REMOTE_URL=https://github.com/secure-device-onboard/epid-verification-service.git
+REMOTE_URL=https://github.com/fido-device-onboard/epid-verification-service.git
 REMOTE_BRANCH=master
```

Comment:
Update the BASE URL in https://github.com/secure-device-onboard/epid-verification-service/blob/master/download-cryptomaterials.sh#L12

to `BASE_URL="http://verifier.fdorv.com"`